packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -437,14 +439,19 @@ export function mapCustomChainToNetworkData(chain: ChainWithRpcUrl) {
   explorerUrls[chain.chainID] = chain.explorerUrl
 }
 
-function isL1Chain(chain: L1Network | L2Network): chain is L1Network {
+export function isL1Chain(chain: L1Network | L2Network): chain is L1Network {
   return !chain.isArbitrum
 }
 
 function isArbitrumChain(chain: L1Network | L2Network): chain is L2Network {
   return chain.isArbitrum
 }
 
+export const TELEPORT_ALLOWLIST: { [id: number]: number[] } = {
+  [ChainId.Ethereum]: [1380012617, 70700], // Rari and PopApex
```

Comment:
Since we removed Orbit chain IDs from `ChainId` enum we're getting more and more cases where we hardcode these IDs. It could get quite messy and out of control.

Thinking if we should create an issue to create a script that generates these nums based on Orbit JSON configs.